article/sections/_4_graph_construction.tex

```diff
@@ -1,3 +1,90 @@
+We call these variables, which cause our outcome and current variables in the iteration, confounding variables.
+We will add each of them to our causal graph.
+
+For the last set of variables, we should explicitly consider the role of time, even in research that may be cross-sectional due to the data that is available to us.
+In reality, how do we think our system evolves over time?
+If we consider multiple observations of a given decision maker, how do variables observed of that decision maker at time $t$ partially cause future variables important to the context or outcomes for that decision maker at time $t+1$?
+How do the actions of a decision maker $i$ at time $t$ partially cause the future context or outcomes of a decision maker $j$?
+We should add explicit nodes to our graph, subscripted or denoted by time, to show the cross-time causal relationships in our system.
+
+At this point, we have added all the outcome, explanatory, effect modifying, mediating, confounding, and time-indexed variables that we can think of to our causal graph.
+They are all disconnected nodes, singletons, in the graph.
+We now focus on pruning nodes from this graph, before drawing our final hypothesized connections.
+In particular, we focus on pruning ``post-outcome'' variables that are not part of the causal graph for future time periods or other observations.
+The reason for this is that conditioning on such post-outcome variables biases one's causal effect estimates.
+
+To remove the problematic variables, we iterate through each of the non-outcome variables in our graph, and we assess whether the variable is actually a result of the outcome (perhaps in combination with other variables in our graph).
+These post-outcome variables temporally follow the outcome variable(s) but do not cause variables in the causal graph for other observations.
+We remove all such post-outcome variables from our graph.
+
+Now is a good time to step back and consider what other researchers have thought.
+Specifically, we should conduct a literature review to see how other researchers have conceptualized the problem that we are working on.
+Have they included variables that we have not?
+Were those variables related our outcomes of interest?
+If so, should we add these variables to our causal graph, and how?
+Do the included variables of other researchers point suggest the existence of confounders in their work that we should include in our graph?
+Have other researchers ascribed differing roles to our graph's current variables than we have?
+For example, have other researchers judged a variable to be confounder, when we solely thought of the variable as an affect modifier?
+Critically examine the evidence for these alternative decisions to see if we should also reconsider how we're judging our variables.
```

Comment:
```suggestion
Now is a good time to step back and consider what other researchers have thought about the problem of interest.
Specifically, we should conduct a literature review to see how other researchers have conceptualized the problem that we are working on.
Have they constructed causal graphs to clearly represent the assumptions of their analysis?
Have they considered or included variables that we have not?
Were those variables related to our outcomes of interest?
If so, should we add these variables to our causal graph?
How should such variables enter our causal graph?
Do the included variables of other researchers suggest the existence of confounders in their work that we should include in our graph?
Have other researchers ascribed differing roles to our graph's current variables than we have?
For example, have other researchers judged a variable to be confounder, when we solely thought of the variable as an affect modifier?
As we answer these questions, and any others that may arise as a result of exploring the literature, we should critically examine the evidence for these alternative decisions to see if we should also reconsider how we are judging our variables and including them in our graph.
```

Reply:
Mo! Question about line 60: "Have they constructed causal graphs to clearly represent the assumptions of their analysis?" After considering the other questions in the list, what do we learn specifically from knowing that they constructed a causal graph for their analysis?

My hope with the list was to capture the common things one would want to learn from examining the causal graph from others, if they have one. Then if I specifically added those questions, I wouldn't need to necessarily be concerned with whether the papers I'm reading construct a causal graph. If I abstract/extract the core of the questions, then they should hopefully be answerable even if a researcher is operating from a potential outcomes framework and didn't draw any causal graph.

Reply:
Well the reason I had in there is to include cases/studies where other researchers haven't even bothered to put together causal graphs at all.